backend/app/recommender/router_utils/recommendations.py

```diff
@@ -78,27 +77,28 @@ async def get_recommended_items(client: AsyncClient, uuids: list[str]):
 # pylint: disable=unused-argument
 @alru_cache(maxsize=512)
 async def get_fixed_recommendations(
-    panel_id: RecommendationPanelId, count: int = 3
+    collection: Collection, count: int = 3
 ) -> list[str]:
     rows = 100
-    if panel_id == "data-source":
-        panel_id = "data source"
-    if panel_id == "all":
-        panel_id = "publication"
-    fq = [f'type:("{panel_id}")']
-    async with httpx.AsyncClient() as client:
-        response = await search(
-            client,
-            ALL_COLLECTION,
-            q="*",
-            qf="id",
-            fq=fq,
-            sort=["id desc"],
-            rows=rows,
-        )
-    if response.status_code != HTTP_200_OK:
+    if collection == "data_source":
```

Comment:
I think consts.py should have variables for each collection name. So here DATA_SOURCE="data_source" in consts.py and import.